services/order.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from typing import List, Dict
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: List[Dict[str, int]],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
```

Comment:
I wrote just variable "date" here, and code still works too, so if this changing won't cause errors, You can try do it too)